index.rst

```diff
@@ -323,69 +331,67 @@ Tag Format
 ^^^^^^^^^^
 
 The tag format is exactly that of a release format, with an additional
-underscore-separated component, ``rc[number]``.  Cycle and rest are
-permitted.
+underscore-separated component, ``rc[number]``.
+RSP build version, cycle and rest are permitted.
 
 Display Name
 ^^^^^^^^^^^^
 
 The display name resembles a Release version, except that it begins with
 "Release Candidate"; the additional component will be appended with a
-dash (to match the semantic version string).  ``r22_0_0_rc1`` will
-have the display name ``Release Candidate r22.0.0-rc1``.
+dash (to match the semantic version string).
+``r22_0_0_rc1`` will have the display name ``Release Candidate r22.0.0-rc1``.
 
 Semantic Version
 ^^^^^^^^^^^^^^^^
 
 The primary components of the version are the same as release: major,
 minor, patch (in general, patch will be ``0`` because it will be a
 prerelease).  ``rc[number]`` will be used as the prerelease (rather than
-the build) field.  Thus, ``r22_0_0_rc1`` will have the version
-``22.0.0-rc1``, and ``r22_0_0_rc1_c0020.003_20210609`` would have the
-version ``22.0.0-rc1+c0020.003.20210609``.
+the build) field.
+Thus, ``r22_0_0_rc1`` will have the version ``22.0.0-rc1``, and ``r22_0_0_rc1_c0020.003_20210609`` would have the version ``22.0.0-rc1+c0020.003.20210609``.
 
 Experimental
 ------------
 
 Experimental tags are used mostly by people working on the Lab machinery
 itself (which is to say, mostly the author of this technote at this
-point).  They start with ``exp_`` and that's really all you can say
-about them (but see below).
+point).
+They start with ``exp_`` and that's really all you can say about them (but see below).
```

Comment:
It does actually matter that the rest of the version has a valid version format, because then it will be parsed and displayed with a proper display name, so that should be done where possible.